docs/changes/io.fits/11913.bugfix.rst

```diff
@@ -0,0 +1 @@
+Ensure masked times round-trip to fits, even if multi-dimensional.
```

Comment:
It's `FITS` not `fits`.

Reply:
Too much FITS does round-trip to fits... 😉 (j/k j/k)